digdag-docs/src/operators/redshift.md

```diff
@@ -231,3 +231,12 @@ When you set those parameters, use [digdag secrets command](https://docs.digdag.
   password_override: another_password
   ```
 
+* **status_table_cleanup**: TIME VALUES
+
+  Specifies the period of time to clean up the status_table." When "strict_transaction: true" (default), the status_table will be created. status_table will be deleted when the status_table_cleanup period expires and the redshift operator is executed. *Default*: `24h`(24 hours).
```

Comment:
> " When "strict_transaction: true" (default)

The first quotation should be removed?

Reply:
Sorry for the late. That Is unnecessary.

Reply:
Sorry for waiting. I committed the fixed md.